terraform/projects/app-ecs-nodes/main.tf

```diff
@@ -0,0 +1,185 @@
+/**
+* ## Project: app-ecs-nodes
+*
+* Create ECS worker nodes
+*
+*/
+
+variable "additional_tags" {
+  type        = "map"
+  description = "Stack specific tags to apply"
+  default     = {}
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "ecs_image_id" {
+  type        = "string"
+  description = "AMI ID to use for the ECS nodes"
+  default     = "ami-2d386654"
+}
+
+variable "ecs_instance_type" {
+  type        = "string"
+  description = "ECS Node instance type"
+  default     = "t2.medium"
```

Comment:
What should this be?

Reply:
The instance type will come out of the capacity planning trello story. If you're going to point this at DGU then you'll want at least as big as you currently have.

Reply:
Cool, will bump up 👍